test/sharness/t0030-mount.sh

```diff
@@ -20,14 +20,10 @@ test_launch_ipfs_daemon
 
 # run this mount failure before mounting properly.
 
-test_expect_failure "'ipfs mount' fails when no mount dir (issue #341)" '
-	test_must_fail ipfs mount -f=not_ipfs -n=not_ipns >actual
-'
-
-test_expect_failure "'ipfs mount' looks good when it fails (issue #341)" '
-	! grep "IPFS mounted at: $(pwd)/ipfs" actual >/dev/null &&
-	! grep "IPNS mounted at: $(pwd)/ipns" actual >/dev/null ||
-	test_fsh cat actual
```

Comment:
I think the idea to have two tests is because the first test just checks that the exist code is what we expect and the second one tests that the output is what we expect.

Here the first one "'ipfs mount' fails when no mount dir" would just check that the exit code is not zero. And the second one should be renamed "'ipfs mount' output looks good when it fails" to make clear that it's about the output.

This is a bit different than in git where the exit code and the output are checked in the same test, but I think it makes it easier to debug.